en/server-side/nodejs.md

```diff
@@ -26,6 +26,20 @@ This should print the version number of Node.js like this:
 ```bash
 v20.7.0
 ```
+## Running the Node.js REPL
```

Comment:
Nice work @Arnalv . What do you think of below content?

REPL is a command line interface that stands for Read Evaluate Print Loop, which enables you to run code from the terminal. To get started, type the following command into your terminal: 

```bash
node
```

Once you enter this prompt, you can run code, like this:

```js
> console.log("Hello World")
```

Reply:
Looks good to me

Reply:
Is it possible to update the PR with that content?

Reply:
Yes